### PR TITLE
Registering Pooja Kelkar

### PR DESCRIPTION
**Pull request by poojakelkar**
Hi, Maintainer

You're doing good job, please accept my request..
Thank you..

### CODE REVIEW

File: content/participants/ttjc@poojakelkar/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Pooja Kelker
+type: "participant"
+name: Pooja Kelkar"
+
```
Comment (khusharth) [inline content/participants/ttjc@poojakelkar/index.md:5]: Hey! @poojakelkar don't remove the **portfolio** from here. If you dont have one right now you can just leave it blank like:
* portfolio: ""

File: content/participants/ttjc@poojakelkar/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Currently, a student pursuing B.Tech. Information Technology at SVKM's IOT Dhule,
+Passionate about participating in competitions like Hackathons and organizing events.Keen interest in research on new technologies.
+I like contributing to the community and share my knowledge.
+
+## why are you taking this challenge?
+
+I need a job in December 2020.
+I want to learn.
+I want get mentorship from industry expert and develop skills
```
Comment (khusharth) [inline content/participants/ttjc@poojakelkar/index.md:24]: I want **to** get mentorship ...

Comment (poojakelkar) [reply content/participants/ttjc@poojakelkar/index.md]: i have made the changes please accept my PR

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Comment by gkanishk** [comment]
Welcome on board!!

**Pull request merged**